concepticondata/conceptlists.tsv

```diff
@@ -1,6 +1,6 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
-Sidwell-2021-116	Sidwell, Paul and Alves, Mark	2021		116	basic	English	Vietic		SidwellAlves2021		This concept list is taken from a dataset on Nicobarese languages. The authors have also indicated whether the concepts recur in [Swadesh's list of 100](:ref:Swadesh-1955-100) and [Leipzig-Jakarta list](:ref:Tadmor-2009-100).		
-Sidwell-2021-100	Sidwell, Paul	2021		100	basic	English	Nicobarese		Sidwell2021		This concept list is taken from a dataset on Nicobarese languages. The author has also indicated whether the concepts recur in [Swadesh's list of 100](:ref:Swadesh-1955-100) and [200 items](:ref:Swadesh-1955-100).		
+Sidwell-2021-116	Sidwell, Paul and Alves, Mark	2021		116	basic	English	Vietic		Sidwell2021b		This concept list is taken from a dataset on Vietic languages. The authors have also indicated whether the concepts recur in [Swadesh's list of 100](:ref:Swadesh-1955-100) and the [Leipzig-Jakarta list](:ref:Tadmor-2009-100).		
+Sidwell-2021-98	Sidwell, Paul	2021		98	basic	English	Nicobarese		Sidwell2021		This concept list is taken from a dataset on Nicobarese languages. The author has also indicated whether the concepts recur in [Swadesh's list of 100](:ref:Swadesh-1955-100) and [200 items](:ref:Swadesh-1955-100).		
```

Comment:
Please do not remove the items completely. Only remove the values for `CONCEPTICON_GLOSS` and `CONCEPTICON_ID` for the two concepts that have been discussed.